src/backend/marsha/account/templates/account/password_reset_confirm.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'account/base.html' %}
+{% load i18n static %}
+
+{% block content %}
+
+<div id="content-main" class="inner-container">
+  {% if validlink %}
+    <p>{% translate "Please enter your new password twice so we can verify you typed it in correctly." %}</p>
```

Comment:
```suggestion
    <p>{% translate "Please enter your new password twice so we can verify you typed it correctly." %}</p>
```
Not sure but I think you can remove `in`

Reply:
This also comes from Django ^^ I'll keep it like this then.